docs/docs/quickstart/complete-guide.mdx

```diff
@@ -374,13 +374,13 @@ docker compose -f docker-compose-scripts/docker-compose.yml up -d execution cons
 ? Run the script with the selected services execution, consensus? [Y/n]
 ```
 
-Notice that it says "Run the script with the selected services **execution, consensus**?". This is because by default only this nodes will be up and running at the beginning, as they need to sync with the network first and these can take days.
+Notice that it says "Run the script with the selected services **execution, consensus**?". This is because by default only these nodes will be up and running at the beginning, as they need to sync with the network first and these can take days.
```

Comment:
```suggestion
Notice that it says "Run the script with the selected services **execution, consensus**?". This is because by default only these nodes will be up and running at the beginning, as they need to sync with the network first and this can take days.
```